module-4/backend/week_two.md

```diff
@@ -1,22 +1,57 @@
 ## Week Two - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
 
-Note: When you're done, submit a PR. 
 
 1. What's one difference between ES5 and ES6?
+  Syntax in various areas.
+  ;'s not required.
+  explicit returns not required. 
+  
 2. What's the difference between asynchronous and synchronous JavaScript? 
+  synchronous JavaScript, which is JS's natural behavior, is processing functions or executable code in the specified order and never breaking order.
+  aysynchronous JS, which has special requirements with either engines or libraries, which allows it to run is taking applicable code and running multiple functions or modules of code at the same time. 
+  
 3. What are the four pillars of Object Oriented programming?
+
+  abstraction--> Hiding parts of one module of code from another while still exposing the same code to see what it only needs.
+  polymorphism--> Allowing sets of code to work and behave like the parent or base code of them.  SImilar with inheritance
+  encapsulation--> Hiding data and inner workings of one set of code from another set of code.
+  inheritance--> Allowing different modules of code that have different responsibilites, to act or behave in the same manner and produce the same desired result. 
+  
```

Comment:
This definition could be a little sharper - I'd mention child classes deriving from parent classes that inherit all the functionality of the parent class